.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ on:
   workflow_dispatch:
 
 env:
-  flutter_version: "3.16.0"
+  flutter_version: "3.19.3"
```

Comment:
Is this required for web 0.51 or is this change out of scope?

Reply:
`package:web 0.3.0` is not supported on Flutter 3.16.0.

Reply:
I set it to the stable cherry picks Flutter version I'm using but feel free to set to any lower stable version that works.

Reply:
I think we should then increase the min version inside pubspec as well
Currently it's still `flutter: ">=3.16.0"`

Reply:
Done. It's not important for the Dart tool but it does add some clarity for humans. Well spotted!